bluetooth-audio/start.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+if [[ "$DISABLE_BLUETOOTH" ]]; then
```

Comment:
I'd rather use `-z` option, I think the intent is more obvious to the reader. Also lets you remove the `"` so you can do: `if [[ -z $DISABLE_BLUEETOOTH ]];`. But it's personal preference maybe.

Reply:
will add

Reply:
@tmigone did you mean: `if [[ ! -z $DISABLE_BLUEETOOTH ]];`? You said: `if [[ -z $DISABLE_BLUEETOOTH ]];`

Reply:
Yeah sorry, `-z` would be if the variable is not defined. I'm making all kinds of mistakes today! You can do then`-n` instead of `-z`.